app/Jobs/Emails/PresentationSubmissions/SelectionProcess/PresentationSpeakerSelectionProcessEmail.php

```diff
@@ -42,6 +43,13 @@ public function __construct
         $payload['summit_site_url'] = $summit->getDefaultPageUrl();
         $payload['speaker_full_name'] = $speaker->getFullName();
         $payload['speaker_email'] = $speaker->getEmail();
+
+        $test_email_recipient = EmailTest::getEmailAddress();
+
+        if (!is_null($test_email_recipient)) {
```

Comment:
isnt better to test by empty ( that includes null ) ?

Reply:
done